lib/core/pagination/pagination.component.scss

```diff
@@ -96,6 +77,10 @@
             max-height: 250px !important;
         }
 
+        &.adf-pagination__empty {
```

Comment:
do we need it? hasItems in the template already hide it

Reply:
That's what I also thought in the beginning, but... if we do not set 'display:none;', then the pagination is still displayed with its border-top: 1px solid rgba(0, 0, 0, 0.07) having height: 48px;
On demo-shell is not noticeable, but on ACA it looks really ugly.

Reply:
@eromano @suzanadirla maybe we take a chance fixing this in the ADF right now, and have proper pagination hiding on empty?

Reply:
I tried to find a better way, but I do not see much options: 
As solution no 2, we could add instead a wrapping div that would have the styles applied to it, which would be hidden along with any border, height, spacings applied to it - and the component tag would remain completly unstyled.

What ever is the chosen solution, we should make sure that all components that have all content inside `<ng-container *ngIf="` element would get the empty state fixed - so no spacing or borders/background would still be displayed when they should not.

Reply:
After talking with Denys, I've changed the styling on the empty pagination.
I also added 2 variables so the styling for the empty state can be changed (on ACA side, for example).
@eromano, please check. Hope it's better now.